integration-tests/pom.xml

```diff
@@ -93,7 +93,39 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <excludedGroups>RealServiceOnly</excludedGroups>
+                </configuration>
+                <executions>
+                    <execution>
+                        <id>mock-tests</id>
+                    </execution>
+                </executions>
+            </plugin>
         </plugins>
     </build>
 
+    <profiles>
+        <profile>
+            <id>functionalTests</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <configuration>
+                            <excludedGroups>MockOnly</excludedGroups>
+                        </configuration>
+                        <executions>
+                            <execution>
+                                <id>real-tests</id>
+                            </execution>
```

Comment:
Won't this run all the tests again?Don't we just want to run the functional test ones in this execution?

Reply:
true, I haven't thought it through.

Looks like there needs to be another profile for mock only and all other tests could be executed in the "normal" execution. I'll update the PR

Reply:
I would view all other tests, including mock as the default. Perhaps just run the real tests in the separate profile ?

Reply:
Now by default all tests by the real service tests are executed, and if switched to `functionalTests`, only the tests that work well with mocks are excluded.